### PR TITLE
Rename Flags.getFeature() -> Features.isEnabled()

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Rename Flags -> Features

## Justify why they are needed

Make names reflect actual usage

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2275** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2275" target="_blank">Graphite</a> 
    * **PR #2277** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2277" target="_blank">Graphite</a> 
      * **PR #2278** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2278" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2278?utm_source=stack-comment).

File: apps/store/src/services/Features.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as process from 'process'
 
-export const Flags = {
-  getFeature(feature: FeatureFlag) {
+export const Features = {
+  isEnabled(feature: FeatureFlag) {
```
Comment (robinandeer) [inline apps/store/src/services/Features.ts:4]: nit: or just `Features.enabled(...)`

`isEnabled` is nice for variable names but not 100% needed here I think

**Review by robinandeer — APPROVED** [review]
I like it!

**Comment by alebedev** [comment]
@alebedev queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2278).

**Comment by alebedev** [comment]
[Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2278) rebased this pull request as part of a merge.

**Pull request merged**